atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -438,7 +438,8 @@ private TransactionManager serializableInternal(@Output List<AutoCloseable> clos
                         sweepStrategyManager,
                         follower,
                         instrumentedTransactionManager,
-                        persistentLockManager),
+                        persistentLockManager,
+                        runBackgroundSweep()),
```

Comment:
nit: `runBackgroundSweepProcess()`, to avoid confusion with sweep being enabled?